docs/tasks/debug-application-cluster/audit.md

```diff
@@ -183,31 +225,34 @@ API can be found [here][audit-api] with more details about the exact fields capt
 The behavior of the `--audit-log-path` flag changes when enabling the `AdvancedAuditing`
 feature flag. This includes the cleanups discussed above, such as changes to the `method`
 values and the introduction of a "stage" for each event. As before, the `id` field of
```

Comment:
it reads to me that the `id` field is renamed to `auditID`, based on the sample pasted below?

Reply:
In legacy format it is called "id".
In json format it is called "auditID".

Reply:
Yes. The problem is that when it says "As before, ...", a careful reader will start comparison, and he/she will find the difference.

Reply:
You are right.
These sentences are used to describe the "Legacy format". Moved them to another place.